ld/nodemcu.ld

```diff
@@ -103,7 +103,11 @@ SECTIONS
     *(.init.literal)
     *(.init)
     
-    *sdk/esp_iot_sdk_*lib/lib*.a:*(.literal .text)
```

Comment:
Could I request a comment here stating that we're intentionally excluding `libwps.a` even though it claims to have iram'd functions?